mlflow/utils/annotations.py

```diff
@@ -164,7 +187,7 @@ def wrapper(*args, **kwargs):
             raise TypeError(f"Method {func.__name__} only takes keyword arguments.")
         return func(**kwargs)
 
-    indent = _get_min_indent_of_docstring(wrapper.__doc__)
+    indent = _get_min_indent_of_docstring(str(wrapper.__doc__))
```

Comment:
```suggestion
    indent = _get_min_indent_of_docstring(wrapper.__doc__) if wrapper.__doc__ else ...
```

Reply:
thanks for the catch - missed that update